lib/src/field_mapper.dart

```diff
@@ -36,6 +39,7 @@ class FieldMapper {
     lastName: "DCS",
     middleName: "DAD",
     expirationDate: "DBA",
+    expirationDateAlt: "ZOE",
```

Comment:
@moesaid thanks for adding this. Can you link to the documentation that explains this field, "ZOE"?

Reply:
It seems that my expired licenses ( USA ) consistently return the expiration date using the `DBA` field. However, for newly renewed licenses, the `DBA` field is often empty, and the expiration date is provided under the `ZOE` field instead. I'm not sure if this behavior is specific to my state or part of a broader update to the AAMVA standard. Unfortunately, I couldn’t find any official documentation about this change on aamva.org.